src/helpers/getFilteredTodo.ts

```diff
@@ -0,0 +1,31 @@
+import { Status } from '../types/Status';
+import { StatusSelect, Todo } from '../types/Todo';
+
+interface IQuery {
+  status: Status;
+  query: string;
+}
+
+export const getFilteredTodos = (initialTodos: Todo[], query: IQuery) => {
+  const { status, query: searchQuery } = query;
+
+  // Filter based on status
+  let filteredTodos = initialTodos;
```

Comment:
You don't need let it in this case. You can use const and then return operations with the array.